packages/@react-spectrum/calendar/src/Calendar.tsx

```diff
@@ -24,6 +24,7 @@ import {useProviderProps} from '@react-spectrum/provider';
 function Calendar<T extends DateValue>(props: SpectrumCalendarProps<T>, ref: FocusableRef<HTMLElement>) {
   props = useProviderProps(props);
   let {visibleMonths = 1} = props;
+  visibleMonths = Math.max(visibleMonths, 1);
```

Comment:
When setting visibleMonths in Storybook controls, it defaults to 0 which caused a crash.

Reply:
Good to know. Yeah I would lean towards keeping it since we have similar guardrails elsewhere.